tests/auth/login/google.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Google OAuth Login Flow', () => {
+  test('should login via Google and redirect to home', async ({ page, context }) => {
+    await page.goto('/');
+
+    // Click Google login button (assuming button has id="google-login")
+    const [popup] = await Promise.all([
+      context.waitForEvent('page'),
+      page.click('#google-login'),
+    ]);
+
+    await popup.fill('input[type="email"]', 'testgoogleuser@gmail.com');
+    await popup.click('button:has-text("Next")');
+    await popup.fill('input[type="password"]', 'your-google-password');
```

Comment:
Wouldn't it be better to get the value through config rather than hardcoding it?